kitsune/wiki/tests/test_views.py

```diff
@@ -902,17 +902,17 @@ def test_custom_locale_mapping(self):
         Test that the article is shown in the defined fallback locale
         """
 
-        # Create a document localized into bn-BD. Attempt to resolve to the bn-IN version
+        # Create a document localized into es. Attempt to resolve to the ca version
```

Comment:
Similar let's use either `bn-BD` or `bn-IN` and `bn`.

Reply:
As per our conversation over slack, this feature was implemented for `bn-BD` and `bn-IN` locale fallback. As both locales are merging together, we do not need it anymore.